cint/cint/src/loadfile.cxx

```diff
@@ -559,6 +559,46 @@ int G__isfilebusy(int ifn)
   return(flag);
 }
 
+#if !defined(G__WIN32)
```

Comment:
Isnt't the compilation going to fail on Windows?

Reply:
The calls to `G__cachingstat` below are also in the `#else` case of `#ifdef G__WIN32`, so, no, it shouldn't fail.